airflow/configuration.py

```diff
@@ -252,6 +252,10 @@ def run_command(command):
 # privacy.
 demo_mode = False
 
+# The amount of time (in secs) webserver will wait for initial handshake
+# while fetching logs from other worker machine
+handshake_timeout = 5
```

Comment:
may be just `log_fetch_timeout_sec` as this timeout only impact when log gets fetched from other machines..